application/models/services/CopyQuestion.php

```diff
@@ -93,7 +93,11 @@ public function copyQuestion($copyOptions)
     public function createNewCopiedQuestion($questionCode, $groupId, $questionToCopy)
     {
         $this->newQuestion = new \Question();
-        $this->newQuestion->attributes = $questionToCopy->attributes;
+        // We need to use setAttributes here with $safeOnly=false to avoid issue #18323.
+        // Otherwise validators are loaded before setting the attributes. 
+        // Right now, probably a bug, some rules are added as validators conditionally, depending on the attribute values.
```

Comment:
Not a really bug since (for example) a subquestion can not have title `other` if` other == "Y"` for parent question.

Since we copy here : if you disable validation : you can set same code than another question (for example).

You have to disable validation for question ? Not only subquestion ?

Reply:
Oups … we save with validation after :)

Reply:
Maybe need to check isNew ?